contents/docs/migrate/migrate-to-cloud.mdx

```diff
@@ -82,73 +118,38 @@ Start by logging in to your existing self-hosted instance and navigating to the
 For the configuration:
 - **Host**: either app.posthog.com for US or eu.posthog.com for EU
 - **Project API Key**: the API key for the project in Cloud
+- **Disable GeoIP**: toggle this on to make sure we keep original location rather than updating to the location of your server.
 
 After clicking save and activating it (toggle on the left side), the Replicator will begin forwarding incoming events to Cloud. Note that it may take around 15-20 minutes for the first events to arrive.
```

Comment:
```suggestion
After clicking save and activating it (toggle on the left side), the Replicator will begin forwarding incoming events to your new instance. It may take around 15-20 minutes for the first events to arrive.
```